### DISCUSSION TITLE
nova-2 pt-BR failing to recognize "um" (1)

### DISCUSSION THREAD

**Discussion by pprobst** [deepgram/community#744] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT

### Details

Hello.

I've been testing Nova-2 streaming through the js-live-example and the `mission/transcribe-your-voice-in-realtime` (in the website), but the model is struggling to recognize the number "1" (pronunciation "um").

Related: #574.

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

/v1/listen?encoding=ogg-opus&sample_rate=16000&interim_results=true&punctuate=true&model=nova-2&language=pt-BR

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

3f421b5a-44f8-4d81-bfbf-4c93a50e9ae8

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by RyanChimienti** [comment, marked as answer]
Hey @pprobst , can you try adding `filler_words=true` to your query string? The [filler words feature](https://developers.deepgram.com/docs/filler-words) is set to false by default, which means words like "um" get stripped out.

**Comment by pprobst** [comment]
Thank you @RyanChimienti, It works, but at the cost of having to accept "filler words" even if I don't want them. And "um" is a very important word in Portuguese -- it's not only a numeral but also an indefinite article. In other words, it's like suppressing "an" in English 🫠
I'd consider this a "bug" in the Portuguese model. Would it be possible to add a condition to *not* suppress "um" for pt and pt-BR without having to use `filler_words=true`?

**Reply by RyanChimienti** [reply]
Yeah, that's a very fair point. I relayed this to the team--appreciate the feedback.
